zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -261,6 +261,26 @@ private void broadcast(String noteId, Message m) {
     }
   }
 
+  private void broadcastExcept(String noteId, Message m, NotebookSocket exclude) {
+    synchronized (noteSocketMap) {
+      List<NotebookSocket> socketLists = noteSocketMap.get(noteId);
+      if (socketLists == null || socketLists.size() == 0) {
+        return;
+      }
+      LOG.info("SEND >> " + m.op);
```

Comment:
You'd better change the level of this log to debug. I saw the issue to change it with this kind of logs